templates/shop/order_pay/web_elements/capture.html.twig

```diff
@@ -4,6 +4,6 @@
 
 {% block title %}{{ 'flux_se_sylius_stripe_plugin.gateway_factory.stripe_web_elements'|trans }} | {{ parent() }}{% endblock %}
 
-{% block body %}
+{% block content %}
```

Comment:
This allows to display the default layout header, right ?
Can you add a screenshot to this PR to see what the result will be ?